Handles the special case of a where condition on the table from

shaoshiva opened the pull request:
Handles the special case of a condition in a dedicated query (not a join) whose field is on the table "from" by replacing it with the value of the field property on the model "from".

shaoshiva committed:
Handles the special case of a where condition on the table from

Handles the special case of a condition in a dedicated query (not a join) whose field is on the table "from" by replacing it with the value of the field property on the model "from".

smeeckaert commented on classes/manymany.php:
```diff
@@ -241,8 +241,22 @@ protected function _build_query_where($query, $conditions, $alias_to, $model_fro
+						$condition[0] = \DB::expr(\DB::quote($model_from->{$field}));
+					} else {
+						// Skip if the field property doesn't exist on the $model_from
+						continue;
```
Perhaps it should throw an Exception instead of silently fail. This could help identify typo.

shaoshiva replied:
:+1:

shaoshiva committed:
Throws an exception when a field does not exists on the model from

Throws an exception instead of a silent fail when a field does not exists on the model from

smeeckaert commented:
:+1:

jguyomard commented:
:+1:
